File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        Map<Integer,Integer> mp=new HashMap<>(4);
        for(int i=0;i<numbers.length;i++){
            int tmp=target-numbers[i];
            if(mp.containsKey(tmp)){
                return new int[] {mp.get(tmp),i+1};
            }
            mp.put(numbers[i],i+1);
        }
return new int[0];
        
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int size=nums.size();
        int sum=0;
        for(int i=0;i<size;i++){
            for(int j=i+1;j<size;j++){
                sum=nums[i]+nums[j];
                if(sum==target)
                    return {i,j};
            }
        }
    }
};
